semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/span/SpanSemanticConventionUtils.java

```diff
@@ -58,6 +63,49 @@
     return Optional.of(hostAttribute.getValue());
   }
 
+  /**
+   * @param event Object encapsulating span data
+   * @return URI based on Open Tracing format
+   */
+  public static Optional<String> getURIforOpenTracingFormat(Event event) {
+    String host = SpanAttributeUtils.getStringAttributeWithDefault(
+        event, OT_PEER_HOSTNAME,
+        SpanAttributeUtils.getStringAttribute(event, OT_PEER_IP));
+    if(StringUtils.isBlank(host)) {
+      return Optional.empty();
+    }
+
+    if (SpanAttributeUtils.containsAttributeKey(event, OT_PEER_PORT)) {
+      return Optional.of(String.format(
+          "%s:%s", host, SpanAttributeUtils.getStringAttribute(event, OT_PEER_PORT)));
+    }
+    return Optional.of(host);
+  }
+
+  /**
+   * @param attributeValueMap map of span data and attribute value
+   * @return URI based on Open Tracing format
+   */
+  public static Optional<String> getURIforOpenTracingFormat(Map<String, AttributeValue> attributeValueMap) {
```

Comment:
I think we are not using this function. We can remove this.

Reply:
Removed it.